### DISCUSSION TITLE
Offer ways to configure reverse DNS

### DISCUSSION THREAD

**Discussion by otavio** [mysticaltech/terraform-hcloud-kube-hetzner#223] [Ideas]
On other deployments we've been doing, using standalone servers in Hetzner Cloud, we've done the reverse DNS for the nodes. This would be something we'd like to do for our K8S cluster as well. However, it seems we cannot do it without hacking the module. Are we missing something?

**Comment by mysticaltech** [comment]
Hello @otavio, that is indeed something this module does not do out of the box. Since nodes are given names, it wouldn't be too hard to associate each automatically to a proper FQDN (base domain name) and insert it as [rdns](https://registry.terraform.io/providers/hetznercloud/hcloud/latest/docs/resources/rdns). 

So we could add an optional base domain in the variables, and if given we set the rdns for each node. That seems fully doable! If that is something you are willing to work on, please shoot a PR! :) Otherwise, we may consider adding it ourselves in the future.

**Reply by otavio** [reply]
We are exploring the module and considering we adopt it, we'll contribute for sure.

**Reply by otavio** [reply]
By the way, I created the issue #224 to coordinate it.

**Reply by mysticaltech** [reply]
Good to hear! Let's continue all technical discussions there if needed.

**Reply by mysticaltech** [reply]
Closing as done.

**Comment by mysticaltech** [comment]
Implemented in PR: https://github.com/mysticaltech/terraform-hcloud-kube-hetzner/pull/2075

Closing this discussion as addressed for the v3 workstream.
